sympy/concrete/summations.py

```diff
@@ -483,7 +483,10 @@ def is_convergent(self):
             return S.false
 
         ### ------------- integral test -------------- ###
-        if is_decreasing(sequence_term, interval):
+        inequality = Derivative(sequence_term).doit() <= 0
```

Comment:
Are we sure that `sequence_term` is univariate? If so we can just write `sequence_term.diff() <= 0`